### ISSUE TITLE
Ligand activity score 

### ISSUE THREAD

**Issue by keenhl** [saeyslab/nichenetr#12]
This is my first time using NicheNet and I am trying to get a feel for what constitutes a good score for ligand activity.  For example, in one analysis, I am getting a top Pearson score of around 0.008, while in another analysis around 0.04.   Alternatively, should I concentrate on AUROC, and filter for values > 0.50?

Thanks for your advice.

**Comment by browaeysrobin** [comment]
Hi @keenhl,

I would say that a good Pearson score for top ligands is around 0.10 and higher. 

If you have low non-zero scores (such as 0.04), it might still be that the ranking of ligands is valuable, although this ranking will only be based on a few top predicted target genes of a ligand that are in the geneset of interest. 

Scores around zero (such as 0.008) mean that top predicted targets of ligand are not enriched in the geneset of interest compared to a background, thus that ranking of ligands is not valuable in that case. These ligands will typically have an AUROC around 0.50.

**Comment by keenhl** [comment]
Thanks.  That's very helpful.

**Comment by cvanderaa** [comment]
Thanks for the package! I have 2 follow-up questions regarding the PCC:

1.  I've run a few tests and I always get positive PCC. I see that the PCCs you display in your vignette are also all positives. Wouldn't we expect that for some ligands there might be slight anti-correlation? This could be the case when, for a particular ligand, higher weights are found in the background set than in the gene set of interest compared to what is expected at random. 

2. When looking into your code, I noticed you're computing p-values for the PCC but you don't return them in the final `tibble` object. Couldn't that be useful?

**Comment by browaeysrobin** [comment]
Hi @cvanderaa 

1. For some datasets I worked on, I have encountered some ligands with a negative PCC, but this negative score is always around zero. Because the gene set of interest is typically much smaller than the background, a stronger anti-correlation score for a ligand would only occur when the gene set of interest would almost entirely consist of the genes with the lowest regulatory potential scores for a ligand (but this would only rarely occur).

2. I decided not to include them because these p-values are almost always very low (because the sample size, for this test the number of target genes, is so big)
